### ISSUE TITLE
build_depend tags in rtt_ros_integration test packages

### ISSUE THREAD

**Issue by meyerj** [jhu-lcsr-forks/rtt_ros_integration#42]
The `<build_depend>` tags in `tests/*/package.xml` contain Orocos package names. While this might work locally, rtt_ros_integration will probably not build on the buildfarm. 

All dependencies must be either in the `ROS_PACKAGE_PATH` or in the rosdep database:

```
$ rosdep install --from-path . --ignore-src
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
rtt_roscomm_tests: Cannot locate rosdep definition for [ocl-deployment]
rtt_ros_tests: Cannot locate rosdep definition for [ocl-deployment]
rtt_rospack_tests: Cannot locate rosdep definition for [ocl-deployment]
```

**Comment by jbohren** [comment]
Ah yeah, these should have been removed and replaced with just `ocl`.

**Comment by meyerj** [comment]
This was fixed in a4ad25c451796eecfb2fbb3e2c6cf1562af8b3be.
